src/commands/Suggestions/suggest.ts

```diff
@@ -0,0 +1,99 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed, TextChannel } from 'discord.js';
+import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { GuildSettings } from '@lib/types/settings/GuildSettings';
+import { BrandingColors } from '@utils/constants';
+import { PermissionLevels } from '@lib/types/Enums';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_SUGGEST_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_SUGGEST_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<suggestion:string{3,2000}>'
```

Comment:
```suggestion
			usage: '<suggestion:string{3}>'
```

Max is 2K because of Discord's limit already, surely no need for us to specify it as well.